Wikipedia/mw-bridge/CommunicationBridge.m

```diff
@@ -26,6 +27,7 @@ - (CommunicationBridge*)initWithWebView:(UIWebView*)targetWebView;
 
         __weak CommunicationBridge* weakSelf = self;
         [self addListener:@"DOMContentLoaded" withBlock:^(NSString* type, NSDictionary* payload) {
+            [weakSelf.webView wmf_enableJavascriptToXcodeConsoleLogging];
```

Comment:
@montehurd Does this only log to Xcode, or also the standard console as well?
If so, we probably should only enable this for debug/beta builds.
If not, then we can leave it enabled all the time.